galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.cloud.manager/src/main/java/dev/galasa/cloud/internal/properties/ContainerOverrideImage.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright contributors to the Galasa project 
+ */
+package dev.galasa.cloud.internal.properties;
+
+import dev.galasa.cloud.CloudManagerException;
+import dev.galasa.framework.spi.cps.CpsProperties;
+
+
+/**
+ * Container image override
+ * 
+ * @galasa.cps.property
+ * 
+ * @galasa.name cloud.container.override.TAG.image
+ * 
+ * @galasa.description Overrides the image name used for tagged container 
+ * 
+ * @galasa.required No
+ * 
+ * @galasa.default None
+ * 
+ * @galasa.valid_values A valid image name
+ * 
+ * @galasa.examples 
+ * <code>cloud.container.override.API.image=icr.io/galasadev/demo/api:prod<br>
+ * </code>
+ * */
+public class ContainerOverrideImage extends CpsProperties {
+
+	public static String get(final String tag) throws CloudManagerException {
+		try {
+			return getStringNulled(CloudPropertiesSingleton.cps(), "cloud.container.override." + tag, "image");
```

Comment:
I missing something, why does the property container the term override. I would of thought it would just be `cloud.container.tag.image=xxxxx` which can be overriden like we do with all props

Reply:
thats because the properties you are thinking of are to do with DSE.   This is overridding something specifically coded in the program.   However,  override is a little redundant, so will remove.

Reply:
until we get all the other container providers done,  not sure what capability they can provide.   So we can revisit after ther major 3 have been done.